sbin/submitty_autograding_shipper.py

```diff
@@ -115,60 +118,73 @@ def update_worker_json(name, entry):
+            shutil.move(tmp_json_path, foreign_json)
             print("Successfully updated local autograding_TODO/autograding_worker.json")
-            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID, message="Successfully updated local autograding_TODO/autograding_worker.json")
+            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID,
+                                          message="Successfully updated local autograding_TODO/"
+                                                  "autograding_worker.json")
             return True
         except Exception as e:
-            autograding_utils.log_stack_trace(AUTOGRADING_STACKTRACE_PATH, job_id=JOB_ID, trace=traceback.format_exc())
-            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID, message="ERROR: could not mv to local autograding_TODO/autograding_worker.json due to the following error: "+str(e))
-            print("ERROR: could not mv to local autograding_worker.json due to the following error: {0}".format(e))
+            autograding_utils.log_stack_trace(AUTOGRADING_STACKTRACE_PATH, job_id=JOB_ID,
+                                              trace=traceback.format_exc())
+            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID,
+                                          message="ERROR: could not mv to local autograding_TODO/autograding_worker.json due to the following error: "+str(e))  # noqa: E501
+            print("ERROR: could not mv to local autograding_worker.json due to the following error: {0}".format(e))  # noqa: E501
             return False
         finally:
             os.close(fd)
-    #if we are updating a foreign machine, we must connect via ssh and use sftp to update it.
+    # if we are updating a foreign machine, we must connect via ssh and use sftp to update it.
     else:
-        #try to establish an ssh connection to the host
+        # try to establish an ssh connection to the host
         try:
-            ssh = establish_ssh_connection(None, user, host, only_try_once = True)
+            ssh = establish_ssh_connection(None, user, host, only_try_once=True)
         except Exception as e:
-            autograding_utils.log_stack_trace(AUTOGRADING_STACKTRACE_PATH, job_id=JOB_ID, trace=traceback.format_exc())
-            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID, message="ERROR: could not ssh to {0}@{1} due to following error: {2}".format(user, host,str(e)))
-            print("ERROR: could not ssh to {0}@{1} due to following error: {2}".format(user, host,str(e)))
+            autograding_utils.log_stack_trace(AUTOGRADING_STACKTRACE_PATH, job_id=JOB_ID,
+                                              trace=traceback.format_exc())
+            autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID,
+                                          message="ERROR: could not ssh to {0}@{1} due to following error: {2}".format(user, host, str(e)))  # noqa: E501
```

Comment:
We are trying to move away from `.format`, so if a `.format` makes a line to long, absolutely convert it to a `fstring`.